packages/button/src/LionButton.js

```diff
@@ -129,9 +122,14 @@ export class LionButton extends DisabledWithTabIndexMixin(
     };
   }
 
+  get _nativeButton() {
```

Comment:
Can we make this _nativeButtonNode for consistency in Subclasser apis?
Think we agreed on this a few weeks ago :)

Reply:
changed 👍